src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -514,14 +514,42 @@ export async function getFeatureIdType(output: Log, env: NodeJS.ProcessEnv, user
 	}
 }
 
+export function getBackwardCompatibleFeatureId(id: string) {
+	const migratedfeatures = ['aws-cli', 'azure-cli', 'desktop-lite', 'docker-in-docker', 'docker-from-docker', 'dotnet', 'git', 'git-lfs', 'github-cli', 'java', 'kubectl-helm-minikube', 'node', 'powershell', 'python', 'ruby', 'rust', 'sshd', 'terraform'];
+	const renamedFeatures = new Map();
+	renamedFeatures.set('golang', 'go');
+	renamedFeatures.set('common', 'common-utils');
+
+	const newFeaturePath = 'ghcr.io/devcontainers/features';
+	// Note: Pin the versionBackwardComp to '1' to avoid breaking changes.
+	const versionBackwardComp = '1';
+
+	// Mapping feature references (old shorthand syntax) from "microsoft/vscode-dev-containers" to "ghcr.io/devcontainers/features"
+	if (migratedfeatures.includes(id)) {
+		return `${newFeaturePath}/${id}:${versionBackwardComp}`;
```

Comment:
We are using `@` as the separator (unless this changed, but the parsing below still uses that).

Reply:
According the [spec](https://github.com/devcontainers/spec/blob/main/proposals/devcontainer-features.md#referencing-a-feature) changes for OCI, we now use `:` 😄 

> `<oci-registry>/<namespace>/<feature>[:<semantic-version>]`
> eg. ghcr.io/user/repo/go:1

Reply:
We're trying to conform to the OCI spec for identifiers (you should be to just do an `oras pull <identifier>`).  

Tags are preceded by a `:`.  One thing we _don't_ support right now is `...@<digest>`, which we should do and update the spec accordingly.

Reply:
Sounds good, thanks.